SR2020/2020-03-16-competition-postpone.md

```diff
@@ -0,0 +1,32 @@
+---
+to: sr2020-teams
+subject: Postponing Student Robotics 2020
+---
+
+Hello all,
+
+Following guidance from the UK government and the University of Reading, we will need to postpone Student Robotics 2020 from the original date in April, until some time late June, early July.
+
+## Competition
+
+Please fill in the below doodle poll to let us know your preferred dates.
+
+Doodle Poll: https://doodle.com/poll/rmdg8waeezbdqbz9
+
+The event will still take place over a weekend, at the University of Reading.
+
+We hope to confirm the updated date in the next few weeks, so please let us know as soon as you can.
+
+## Tech Days
+
+There are no current planned tech days. Once things have settled down, we can organise some.
+
+The [forum][forum] is still a great place to get help from blueshirts and other teams, in addition to showing off your progress!
+
+If you are looking for more in-depth support from blueshirts, you can [email us][teams-email] to request remote mentoring.
+
+
+If you have any further questions, please [let us know][teams-email].
```

Comment:
[NIT] we should probably make the address visible. just so people can see it without launching their email app, 

```suggestion
If you have any further questions, please [let us know by emailing teams@studentrobotics.org][teams-email].
```
(Same with usage above)